webui/lib/api/schema/swagger.json

```diff
@@ -1467,7 +1467,9 @@
                 6,
                 7,
                 8,
-                9
+                9,
+                10,
+                11
             ],
```

Comment:
these files are supposed to be auto generated using `make generate-swagger`. Though it seem we have an issue with the command and the fix is to remove `pkg/types` from
https://github.com/bacalhau-project/bacalhau/blob/85cbe5b3fa662191c19b2599ee32232a89c6753f/scripts/generate_swagger.sh#L8-L15

The flow is:
```
### generate new swagger schema
 make generate-swagger

### update webui generated API code
cd webui && yarn run generate-api 
```

We need to automate some of these steps, but we already have a [github workflow](https://github.com/bacalhau-project/bacalhau/actions/runs/11709781206/job/32614574075?pr=4698) that verifies swagger.json is up to date. It is currently failing, but the workflow isn't! @jamlo is the GithubActions expert and might know why.

For this PR, just auto-generate swagger and don't worry about the workflow

Reply:
This PR fixes swagger generation https://github.com/bacalhau-project/bacalhau/pull/4699